### PR TITLE
Decorations font and color customization

### LINKED ISSUE #6: Allow customisation of decorations bg/fg decoration colours
e.g background #00CC60, foreground #000000

### PR DESCRIPTION
**Pull request by stormherz**
Added support for font and color customization. There is a small issue with pixel-perfect rendering of the decorations (which isn't quite noticeable and won't affect functionality of the extension), but that's how vscode is currently implemented, can't really do anything about it. Should resolve #6.

### CODE REVIEW

**Comment by capaj** [comment]
@wmaurer is this PR problematic in some way? How is it not merged yet? Customizing of the font is number 1 requested feature in the issues.

**Comment by wmaurer** [comment]
Hi @stormherz & @capaj 
I'm very sorry I've been unresponsive about this. My open-source projects have taken a low priority in the last few months due to personal reasons.

Many thanks @stormherz for implementing this functionality. It looks good from a code point of view.

The only problem I see at the moment is that with decorations for words in the first column. Unfortunately it's a little cut off on the left:
image
My feeling is that we need to position the decorator a little bit to the right ... it's rather difficult, I'd spent quite a bit of time getting the right dimensions and position for the decorator based on the hardcoded font size, but now with it being configurable, it's even harder to get right.

**Comment by stormherz** [comment]
Ah, I see, I'll try to fix things up

**Comment by stormherz** [comment]
@wmaurer could you take a look? I've moved decorators a few pixels to the right, should do the trick for a beam cursor.
That's still a hardcode tweaking, but the main issue is obviously with the decorator API itself, which is quite poor at the moment, so we must use SVG to present jump markers.
If there was a way of overlaying a decorator with a custom content/htm/css =(

**Comment by JGH1000** [comment]
Would be great if this PR was integrated, the current font is really small..

**Comment by seanyu4296** [comment]
Would be great to see this merged

**Comment by olso** [comment]
@wmaurer ❤️ pls

File: package.json [hunk 1]
```diff
@@ -64,6 +64,42 @@
...
+                "default": "black",
+                "description": "Text color of Jumpy decoration in dark themes"
+            },
+
+            "jumpy.lightThemeBackground": {
+                "type": "string",
+                "default": "black",
+                "description": "Background of Jumpy decoration in light themes"
+            },
+
```
Comment (andys8) [inline package.json]: I'd remove the empty lines in `json`.

Comment (stormherz) [reply package.json]: Don't bother, repository is dead

Comment (andys8) [reply package.json]: Looks like that. I'm started to use Acejump. The overall implementation is great. So I can recommend that extension.

https://marketplace.visualstudio.com/items?itemName=lucax88x.codeacejumper

**Comment by olso** [comment]
Holy shhhh 😍😍

**Pull request merged**